src/utils.js

```diff
@@ -661,6 +661,23 @@ go.utils = {
             });
     },
 
+    post_nursereg: function(contact, im) {
+        var payload = {
```

Comment:
Will deal with this with change handling - will need to implement changes in django as well.